src/tests/ftest/util/dfuse_utils.py

```diff
@@ -0,0 +1,228 @@
+                int(pool.pool.svc.rl_ranks[index])
+                for index in range(pool.pool.svc.rl_nr)]])
+        self.svcl.update(svcl, "svcl" if display else None)
+
+    def set_dfuse_cont_param(self, cont, display=True):
+        """Set dfuse cont param from Container object
+
+        Args:
+            cont (TestContainer): Daos test container object
+            display (bool, optional): print updated params. Defaults to True.
+        """
+        self.cuuid.update(cont, "cuuid" if display else None)
+
+    def create_mount_point(self):
+        """Create dfuse directory
+
+        Raises:
+            CommandFailure: In case of error creating directory
+        """
+
+        dir_exists, _ = general_utils.check_file_exists(
+            self.hosts, self.mount_dir.value, directory=True)
+        if not dir_exists:
+            cmd = "mkdir -p {}".format(self.mount_dir.value)
+            ret_code = general_utils.pcmd(self.hosts, cmd, timeout=30)
+            if 0 not in ret_code:
+                error_hosts = NodeSet(
+                    ",".join(
+                        [str(node_set) for code, node_set in ret_code.items()
+                         if code != 0]))
+                raise CommandFailure(
+                    "Error creating the {} dfuse mount point on the following "
+                    "hosts: {}".format(self.mount_dir.value, error_hosts))
+
+    def remove_mount_point(self):
+        """Remove dfuse directory
+
+        Raises:
+            CommandFailure: In case of error deleting directory
+        """
+
+        dir_exists, _ = general_utils.check_file_exists(
+            self.hosts, self.mount_dir.value, directory=True)
+        if dir_exists:
+            cmd = "rm -rf {}".format(self.mount_dir.value)
+            ret_code = general_utils.pcmd(self.hosts, cmd, timeout=30)
+            if 0 not in ret_code:
+                error_hosts = NodeSet(
+                    ",".join(
+                        [str(node_set) for code, node_set in ret_code.items()
+                         if code != 0]))
+                raise CommandFailure(
+                    "Error removing the {} dfuse mount point on the following "
+                    "hosts: {}".format(self.mount_dir.value, error_hosts))
+
+class Dfuse(DfuseCommand):
+    """Class defining an object of type DfuseCommand"""
+
+    def __init__(self, hosts, attach_info):
+        """Create a dfuse object"""
+        super(Dfuse, self).__init__("/run/dfuse/*", "dfuse", hosts)
+
+        # assign attatch info path
+        self.attach_info = attach_info
+
+    def run_dfuse(self, basepath):
+        """ Run the dfuse command.
+
+        Args:
+            basepath: path for daos install dir
+        Raises:
+            CommandFailure: In case dfuse run command fails
+        """
+
+        # create dfuse dir if does not exist
+        self.create_mount_point()
+        # obtain env export string
+        env = self.get_default_env(basepath)
```

Comment:
Something else to consider: `ExecutableCommand` has a `env` attribute that is intended to be assigned to an `EnvironmentVariable` and gets included in the `__str__()` output.  The `self.get_default_env()` method could assign (set) the `self.env` instead of being used to get a value.

Reply:
Keeping as it is as per our conversation.